src/components/profile/librariesTable/LibrariesTable.js

```diff
@@ -70,7 +70,7 @@ function TableItem({ agencyName, agencyId }) {
  * @param {obj} props
  * @returns {component}
  */
-export default function LibrariesTable({ data }) {
+export default function LibrariesTable({ data, municipalityAgencyId }) {
   return (
     <>
       <div className={styles.headerRow}>
```

Comment:
her burde man nok også lave det lidt om for at lave tabellen tilgængelig

Reply:
så er det fixet